assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Overview.test.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Overview component tests.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+	render,
+	createTestRegistry,
+	provideModules,
+	provideUserCapabilities,
+	provideUserInfo,
+	provideUserAuthentication,
+	provideSiteInfo,
+	waitFor,
+} from '../../../../../../../tests/js/test-utils';
+import coreModulesFixture from '../../../../../googlesitekit/modules/datastore/__fixtures__';
+import { CORE_MODULES } from '../../../../../googlesitekit/modules/datastore/constants';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { MODULES_SEARCH_CONSOLE } from '../../../datastore/constants';
+import { getWidgetComponentProps } from '../../../../../googlesitekit/widgets/util';
+import Overview from './Overview';
+
+jest.mock( '../../../../../hooks/useBreakpoint', () => ( {
+	BREAKPOINT_SMALL: 'small',
+	useBreakpoint: () => 'tablet',
+} ) );
```

Comment:
Would you please add some context as to why this is needed and why these specific values are provided?

Reply:
Thanks for bringing this up. This has made me take stock, and take a different approach - see https://github.com/google/site-kit-wp/pull/5713/commits/62724fd634bdd46663667efc3dc6f29518eae6d6